### PR TITLE
Restore check-out state gc rule

### PR DESCRIPTION
**Pull request by liayoo**
- Updated check-out history & refund paths
  - /checkout/history/${networkName}/${chainId}/${tokenId}/${ainAddr}/${checkoutId}/data -> /checkout/history/${networkName}/${chainId}/${tokenId}/${ainAddr}/${checkoutId}
  - /checkout/history/${networkName}/${chainId}/${tokenId}/${ainAddr}/${checkoutId}/refund -> /checkout/refunds/${networkName}/${chainId}/${tokenId}/${ainAddr}/${checkoutId}
- Restored check-out state gc rule

### CODE REVIEW

File: common/path-util.js [hunk 1]
```diff
@@ -232,16 +232,10 @@ class PathUtil {
...
-    return CommonUtil.appendPath(
-        PathUtil.getCheckoutHistoryPath(networkName, chainId, tokenId, address, checkoutId),
-        PredefinedDbPaths.CHECKOUT_HISTORY_DATA);
-  }
-
-  static getCheckoutHistoryRefundPath(networkName, chainId, tokenId, address, checkoutId) {
-    return CommonUtil.appendPath(
-        PathUtil.getCheckoutHistoryPath(networkName, chainId, tokenId, address, checkoutId),
-        PredefinedDbPaths.CHECKOUT_HISTORY_REFUND);
+  static getCheckoutHistoryRefundsPath(networkName, chainId, tokenId, address, checkoutId) {
```
Comment (platfowner) [inline common/path-util.js:235]: minor: now 'getCheckoutRefundPath' or 'getCheckoutRefundsPath'?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks for handling this issue!

File: common/path-util.js (hunk 1 above)
Comment (liayoo) [reply common/path-util.js]: Thanks for pointing that out! Updated.

**Comment by liayoo** [comment]
Thanks for the review. Now merging~

**Pull request merged**